README_zh.md

```diff
@@ -2,6 +2,7 @@
 [![Go Report Card](https://goreportcard.com/badge/github.com/kubeedge/kubeedge)](https://goreportcard.com/report/github.com/kubeedge/kubeedge)
 [![LICENSE](https://img.shields.io/github/license/kubeedge/kubeedge.svg?style=flat-square)](https://github.com/kubeedge/kubeedge/blob/master/LICENSE)
 [![Releases](https://img.shields.io/github/release/kubeedge/kubeedge/all.svg?style=flat-square)](https://github.com/kubeedge/kubeedge/releases)
+[![Documentation Status](https://readthedocs.org/projects/kubeedge-anvith-sphinx/badge/?version=latest)](https://kubeedge-anvith-sphinx.readthedocs.io/en/latest/?badge=latest)
```

Comment:
same as previous comment.

Reply:
Same as before. I will remove this for now.

Reply:
Fixed this. Removed the line.